Newsletters: add 226 (2022-11-16)

harding opened the pull request:
- [x] Update releases/RCs, add topic links @harding 
- [x] Client/services section @bitschmidty 
- [x] Core Lightning 5647 @Xekyo (sorry, no Bitcoin Core PRs to summarize this week)

bitschmidty commented on _posts/en/newsletters/2022-11-16-newsletter.md:
```diff
@@ -0,0 +1,189 @@
+
+- **General smart contracts in Bitcoin via covenants:** Salvatore Ingala
+  [posted][ingala matt] to the Bitcoin-Dev mailing list a proposal for a
+  new type of [covenent][topic covenants] (requiring a soft fork) that
```
```suggestion
  new type of [covenant][topic covenants] (requiring a soft fork) that
```

adamjonas replied:
```suggestion
  new type of [covenant][topic covenants] (requiring a soft fork) that
```

LarryRuane commented on _posts/en/newsletters/2022-11-16-newsletter.md:
```diff
@@ -0,0 +1,189 @@
+      previous newsletters), where some amount of fee is paid to
+      forwarding nodes even if a payment fails to reach the receiver.
+      The authors suggest both a *base* upfront fee that's independent
+      from the amount of the payment and a *proportional* fee that
```
```suggestion
      of the amount of the payment and a *proportional* fee that
```

LarryRuane commented on _posts/en/newsletters/2022-11-16-newsletter.md:
```diff
@@ -0,0 +1,189 @@
+      forwarded payments remain pending and the forwarding fees
+      collected.  If a peer's time per fee is high, it considers that
+      peer high-risk and only allows that peer to use a limited number
+      of the local node's slots and funds.  Otherwise it considers the
```
```suggestion
      of the local node's slots and funds.  Otherwise, it considers the
```

LarryRuane commented on _posts/en/newsletters/2022-11-16-newsletter.md:
```diff
@@ -0,0 +1,189 @@
+
+- [Core Lightning #5681][] adds the ability to filter the JSON output of
+  an RPC on the server side.  Filtering on the server side avoids
+  sending unwanted data when using a bandwidth constrained remote
```
```suggestion
  sending unwanted data when using a bandwidth-constrained remote
```

LarryRuane commented on _posts/en/newsletters/2022-11-16-newsletter.md:
```diff
@@ -0,0 +1,189 @@
+
+- [Core Lightning #5698][] updates the experimental developer mode to
+  allow receiving onion-wrapped error messages of any size. BOLT2
+  currently recommends 256 byte errors, but doesn’t forbid longer error
```
```suggestion
  currently recommends 256-byte errors but doesn’t forbid longer error
```

LarryRuane commented on _posts/en/newsletters/2022-11-16-newsletter.md:
```diff
@@ -0,0 +1,189 @@
+- [Core Lightning #5698][] updates the experimental developer mode to
+  allow receiving onion-wrapped error messages of any size. BOLT2
+  currently recommends 256 byte errors, but doesn’t forbid longer error
+  messages and [BOLTs #1021][] is open to encourage use of 1024-byte
```
```suggestion
  messages, and [BOLTs #1021][] is open to encourage use of 1024-byte
```

harding replied:
Edited previous line so this one can stay the same.

LarryRuane commented on _posts/en/newsletters/2022-11-16-newsletter.md:
```diff
@@ -0,0 +1,189 @@
+    rarely happens with legitimate payments.  The second type is *fast
+    jamming* where the slots and funds are blocked only briefly---which
+    happens often with normal payments, potentially making fast jamming
+    harder to mitigate.
```
```suggestion
    harder both to detect and mitigate.
```
(but check my understanding)

harding replied:
I thought about phrasing it like this myself, but none of the solutions for jamming attacks I'm aware of make any attempt at detection.  This is probably important for preserving privacy on onion-wrapped payments.  The solutions focus on trying to increase the cost of unwanted network uses as high as possible while keeping the cost of wanted uses as low as possible.

Leaving this text unchanged.

harding commented:
Pushed edits (thanks @adamjonas, @bitschmidty, and @LarryRuane !), updated the releases/RCs with LND 0.15.5-beta.rc1, and added topic links.  Also reviewed the contributions by @bitschmidty and @Xekyo (thanks!).  Thanks everyone!

azuchi commented on _posts/en/newsletters/2022-11-16-newsletter.md:
```diff
@@ -0,0 +1,198 @@
+  within another offchain construct, such as a payment channel).
+
+    Ingala explains how the work could help design [joinpools][topic
+    joinpools], opportunistic rollups (see [Newsletter #222][news222
```
```suggestion
    joinpools], optimistic rollups (see [Newsletter #222][news222
```

bitschmidty commented on _posts/en/newsletters/2022-11-16-newsletter.md:
```diff
@@ -0,0 +1,198 @@
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [Bitcoin Core 24.0 RC4][] is a release candidate for the
```
I see 24.0 RC4 isnt on bitcoincore.org yet.

bitschmidty commented:
Pushed changes for @azuchi 's suggestion as well as the covenants topic entry text

bitschmidty commented:
Thank you @harding @Xekyo @adamjonas @LarryRuane @azuchi for your contributions this week! 🚀
